tests/test_visitors/test_ast/test_conditionals/test_named_constants.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from wemake_python_styleguide.violations.consistency import (
+    NamedConstantConditionalViolation,
+)
+from wemake_python_styleguide.visitors.ast.conditionals import (
+    NamedConstantConditionalVisitor,
+)
+
+if_statement = """
+variable = 1
+if {0}: ...
+"""
+
+
+@pytest.mark.parametrize('code', [
+    if_statement,
+])
+@pytest.mark.parametrize('comparators', [
+    ('variable < 3'),
+    ('variable'),
+    ('variable is True'),
+    ('variable is False'),
+    ('variable is None'),
+    ('variable is int or not None'),
+])
+def test_valid_conditional(
+    assert_errors,
+    parse_ast_tree,
+    code,
+    comparators,
+    default_options,
+):
+    """Testing that conditionals work well."""
+    tree = parse_ast_tree(code.format(comparators))
+
+    visitor = NamedConstantConditionalVisitor(default_options, tree=tree)
+    visitor.run()
+
+    assert_errors(visitor, [])
+
+
+@pytest.mark.parametrize('code', [
+    if_statement,
+])
+@pytest.mark.parametrize('comparators', [
+    'True',
+    'False',
+    'None',
```

Comment:
We should also test:
1. numbers: `if 0:` and `if 4:` should always be error
2. strings: `if '':` and `if 'test':` should always be error
3. arrays and dicts (both empty and with data, including variables like: `[variable]`)